### ISSUE TITLE
Support GitHub-Flavored Markdown and other extensions

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-render-markdown#2]
https://py-gfm.readthedocs.io/en/latest/

```diff
diff --git a/datasette_render_markdown/__init__.py b/datasette_render_markdown/__init__.py
index 018f433..826b9b5 100644
--- a/datasette_render_markdown/__init__.py
+++ b/datasette_render_markdown/__init__.py
@@ -3,6 +3,7 @@ import bleach
 import markdown
 from datasette import hookimpl
 import jinja2
+from mdx_gfm import GithubFlavoredMarkdownExtension
 
 
 @hookimpl()
@@ -19,7 +20,11 @@ def render_cell(value, column):
 def render_markdown(value):
     html = bleach.linkify(
         bleach.clean(
-            markdown.markdown(value, output_format="html5"),
+            markdown.markdown(
+                value,
+                output_format="html5",
+                extensions=[GithubFlavoredMarkdownExtension()],
+            ),
             tags=[
                 "a",
                 "abbr",
```

**Comment by simonw** [comment]
I'm torn on whether this should be the default behaviour. My use-cases for this plugin all revolve around data pulled from the GitHub API using `github-to-sqlite`.

**Comment by simonw** [comment]
I could have columns ending in `_gfm` use this, while columns ending in `_markdown` or `_md` stick with regular markdown. Then I can add configuration in #1.

**Comment by simonw** [comment]
I can support other extensions too.

**Comment by simonw** [comment]
Example from [the docs](https://python-markdown.github.io/extensions/):
```
markdown.markdown(some_text, extensions=[
    MyExtClass(),
    'myext',
    'path.to.my.ext:MyExtClass'
])
```
So supporting `"extensions": ["myext"]` in metadata configuration should work here.

For GFM markdown I think that will be `"mdx_gfm:GithubFlavoredMarkdownExtension"`.

**Comment by simonw** [comment]
Fixed in #4.

**Comment by simonw** [comment]
Released in [1.1](https://github.com/simonw/datasette-render-markdown/releases/tag/1.1).
